onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -169,6 +173,85 @@ TEST_F(CApiTest, DISABLED_custom_op) {
 }
 #endif
 
+struct OrtTensorDimensions : std::vector<int64_t> {
```

Comment:
Another tensorshape class?

Reply:
Ah, there's another one that works through the C-API?

Reply:
I mean, why do we need this struct? Can we use TensorShape class directly?

Reply:
No, it's meant to be an example of a custom op, which only has access to the C API